src/discord-update-report.js

```diff
@@ -7,8 +7,10 @@ function formatReport(report) {
   out.push('');
   out.push('**CATALOG UPDATE**');
   for (const r of report) {
+    var data = require(`${(r.catalog, { replacement: '-', remove: /[#,.:?()'"/]/g, lower: true }).toLowerCase()}.json`)
```

Comment:
Don't use `var` in modern JS. Here it's a `const`.
Also what if the lookup for the name is not working like an issue in the name? Not sure this is the best implementation